h2o-core/src/main/java/water/fvec/Vec.java

```diff
@@ -394,7 +402,7 @@ public static Vec makeCon(double x, long len, int log_rows_per_chunk, boolean re
 
   public static Vec makeCon(double x, long len, int log_rows_per_chunk, boolean redistribute, byte type) {
     int chunks0 = (int)Math.max(1,len>>log_rows_per_chunk); // redistribute = false
-    int chunks1 = (int)Math.min( 4 * H2O.NUMCPUS * H2O.CLOUD.size(), len); // redistribute = true
+    int chunks1 = (int)Math.min( 4 * H2O.ARGS.nthreads * H2O.CLOUD.size(), len); // redistribute = true
```

Comment:
Good catch!

Reply:
there are a bunch like this in the code base: I'm actually thinking about creating a ticket to review usages of NUMCPUS, Runtime.getRuntime().availableProcessors() and so on